lab_1_classify_by_unigrams/main.py

```diff
@@ -84,6 +128,39 @@ def compare_profiles(
     In case of corrupt input arguments or lack of keys 'name' and
     'freq' in arguments, None is returned
     """
+    """
```

Comment:
just remove if  you do not want to continue for 8

Reply:
OK